File: java/src/P1.java
package bin;

public class P1 {

    public static void main(String[] args) {
        int i = 0;
        int sum = 0;

        for (i = 0; i < 1000; i++) {
            sum += (i % 3 == 0 || i % 5 == 0) ? i : 0;
        }

        System.out.println(sum);
    }

}

File: zig/src/p1.zig
const stdout = @import("std").io.getStdOut().writer();

pub fn main() !void {
        var sum: u32 = 0;
        var i: u32 = 0;

        while (i < 1000) {
                sum += if (i % 3 == 0 or i % 5 == 0) i else 0;
                i += 1;
        }

        try stdout.print("{d}\n", .{sum});
}